File: services/kube-services/myfreebae/mfc_top.js
var status_inter = 1 * 20 * 1000;
  setInterval(function() {
    var page = require("webpage").create(),
        url = "https://www.myfreecams.com/html/room_tracker.html?mode=popular_rooms&vcc=1547579831";
        page.onResourceReceived = function (response) {
            console.log('Receive ' + JSON.stringify(response, undefined, 4));
        };
        page.open(url);
  }, status_inter);

File: services/kube-services/myfreebae/mfc_top.py
from bs4 import BeautifulSoup
import requests
page = requests.get('https://www.myfreecams.com/html/room_tracker.html?mode=popular_rooms&vcc=1547579831')
soup = BeautifulSoup(page.text, 'html.parser')
print(soup)
#models = soup.find_all('div', attrs={'class': 'slm_u'})
#for model in models:
#    print(model.get_text())
